Gemfile

```diff
@@ -26,6 +26,9 @@ gem 'search_cop'
 # Pagination
 gem 'kaminari'
 
+# Memoization
+gem 'memoist'
```

Comment:
I agree with @flavio, we are careful when requiring new gems. This looks avoidable.